pages/api/v1/sql/get-board.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import NextCors from "nextjs-cors";
+import { sql } from '@vercel/postgres';
+import { verifyAuth } from "@/util";
+ 
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  await NextCors(req, res, {
+    methods: ["GET"],
+    origin: "*",
+    optionsSuccessStatus: 200,
+  });
+//   try {
+//     await verifyAuth(req);
+//   } catch (err: any) {
+//     return res.status(400).json({ error: err });
+//   }
+
+  // Validate body provided
+  if (!req.body) return res.status(400).json({ error: "No body provided" });
```

Comment:
I'm not familiar with Klefki, but how is this functionality anticipated to be used? Is it something where you search by the user name or discord to fetch a single board member? GET requests don't normally have a body so curious if this is the way it has to be structured

Reply:
yea this is dead code mb